examples/proxy/record-encryption/cluster-ip/proxy/kustomization.yaml

```
@@ -1,9 +1,14 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: proxy
+namespace: kafka-proxy
 
 resources:
 - proxy-service.yaml
 - proxy-config.yaml
 
+components:
+- ../../kms/vault
```

nit: should it be 
```suggestion
#- ../../kms/vault
```
by default?